ovos_utils/json_helper.py

```diff
@@ -2,7 +2,8 @@
 from ovos_utils.parse import fuzzy_match
 
 
-def merge_dict(base, delta, merge_lists=False, skip_empty=False, no_dupes=True):
+def merge_dict(base, delta, merge_lists=False, skip_empty=False,
+               no_dupes=True, new_only=False):
```

Comment:
I like this, but I'm afraid the signature is becoming heavy with arguments that conflict with each other. Would this be better off as a separate function, like `incorporate_dict()`?

Reply:
im ok with these arguments, since they are optional anyway

i dont think a new func makes that much sense, what this does is merge dictionaries according to params, it would just be confusing to have many func doing the same thing with slightly different names... end users would not now which one to use and probably end up going with this one, but unable to control its behaviour

this is a single operation in my view

Reply:
That's fair.